mailbag/derivatives/pdf.py

```diff
@@ -0,0 +1,43 @@
+import os
+from subprocess import Popen, PIPE
+from mailbag.derivative import Derivative
+
+
+class ExampleDerivative(Derivative):
+    derivative_name = 'pdf'
+
+    def __init__(self, email_account, **kwargs):
+        print("Setup account")
+        super()
+
+    def do_task_per_account(self):
+        print(self.account.account_data())
+
+    def do_task_per_message(self, message, args):
+        wkhtmltopdf = 'wkhtmltopdf.exe'
+        html = message.HTML_Body
+        From_name=message.From.split("<")
+        To_name=message.To.split("<")
+        #htmlnew="<table>  <tr>    <th>From</th>    <th>To</th> </tr>"  + "<tr> <td>"+a+d[1][:-1]+"</td>"+"<td>"+b+c[1][:-1]+"</td>  </tr>"+"</table>"
+        #Adding From and To fields in pdf
+        html_from_to="From :"+"&nbsp"+ From_name[0]+ From_name[1][:-1]+"<br>"+"To:"+"&nbsp"+To_name[0]+To_name[1][:-1]
```

Comment:
I would use a simple `<table><tr><td>To:</td>to_name<td></td</tr></table>` or similar as a table will be easier to edit with CSS. There should also be a way to avoid writing it multiple times. I would try to make a list with all the fields we want and loop though that list so you only have to write the html once. You should be able to use `getattr()` https://stackoverflow.com/questions/3253966/python-string-to-attribute